packages/kit/src/components/SlippageSettingDialog/SlippageSettingDialog.tsx

```
@@ -198,34 +147,38 @@ const SwapsSlippageContentContainer = ({
+                      setSwapSlippageStatus({
+                        key: ESwapSlippageSegmentKey.CUSTOM,
+                        value: item,
+                      });
+                    }}
+                  >{`${item}${
+                    index === swapSlippageCustomDefaultList.length - 1
+                      ? '  '
+                      : ''
+                  }%`}</Button>
+                  {index !== swapSlippageCustomDefaultList.length - 1 ? (
+                    <Divider vertical />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)